contracts/test/ETO/TestNullEquityTokenController.sol

```diff
@@ -25,6 +25,22 @@ contract TestNullEquityTokenController is
 
     }
 
+    bool allowOnTransfer = true;
```

Comment:
please follow code formatting guidelines as found in https://github.com/Neufund/platform-contracts/blob/master/CodeStyle.md

Also look how sections are split in all contracts. We need strong conventions here. Mess is a way to lose money ;>

Reply:
Improved.